app/pages/lab-pages-editor/components/TasksPage/components/EditStepDialog/types/DrawingTask.jsx

```diff
@@ -319,14 +327,30 @@ function DrawingTask({
         </span>
       </div>
       <div className="input-row">
-        <label
-          className="big spacing-bottom-S"
-          htmlFor={`task-${taskKey}-help`}
-        >
-          Help Text
-        </label>
+        <div className="flex-row spacing-bottom-S">
+          <label
+            className="medium"
+            htmlFor={`task-${taskKey}-help`}
+          >
+            Help Text
+          </label>
+          <button
+            aria-label={`Show/Hide Help field`}
```

Comment:
That would change the labels to 'Show help field (closed)' and 'Hide help field (expanded)', since the ARIA expanded state is read out as part of the label. I don't think it's recommended practice to change the labels for toggles,based on the toggle state. Rather, have a single toggle button label that makes sense when accompanied by the open/closed state.